XenAdmin/Commands/MoveVMCommand.cs

```diff
@@ -67,7 +67,7 @@ public MoveVMCommand(IMainWindow mainWindow, VM vm)
         protected override void ExecuteCore(SelectedItemCollection selection)
         {
 
-            if (selection.AllItemsAre<VM>() && new CrossPoolMoveVMCommand(MainWindowCommandInterface, selection).CanExecute())
+            if (selection.Count > 1 && new CrossPoolMoveVMCommand(MainWindowCommandInterface, selection).CanExecute())

```

Comment:
The MoveVMDialog does the intra-pool move of a VM's disks. But if a cross-pool can be done, then we should offer this, even for a single VM.